packages/Benefit/BenefitNoHeading/__tests__/__snapshots__/BenefitNoHeading.spec.jsx.snap

```diff
@@ -29,6 +29,7 @@ exports[`BenefitNoHeading renders with icons set in each item 1`] = `
 
 .c0 > *:not(:last-child) {
   margin-bottom: 1rem;
+  margin-right: 0;
```

Comment:
Most of these snapshot files have a `margin-right: 0` and `margin-bottom: 0`. Let's see if we can have unset styles instead.